### PR TITLE
Fix default value for simple field

### PR DESCRIPTION
**Pull request by GreatWizard**

### CODE REVIEW

**Comment by GreatWizard** [comment]
Fix a missed update on the template for https://github.com/novafloss/formidable-ui/pull/173/files

File: app/templates/components/builder-field-settings.hbs [hunk 1]
```diff
@@ -65,7 +65,17 @@
...
-              {{validated-input label=(t 'settings.default.label') helpText=(t 'settings.default.helpText') model=buffer valuePath='default' maxlength='256'}}
+              <div class="form-group">
+                <div class="col-sm-3 control-label">
+                  <label>
+                    {{t 'settings.default.label'}}
+                  </label>
+                  <p class="text-info">{{t 'settings.default.helpText'}}</p>
+                </div>
+                <div class="col-sm-9">
+                  {{input class='form-control' value=bufferDefault maxlength='256'}}
```
Comment (xcambar) [inline app/templates/components/builder-field-settings.hbs]: Don't you want to favor ember-one-way-control here?

Comment (GreatWizard) [inline app/templates/components/builder-field-settings.hbs]: I prefer do to that in another PR, more globally

Comment (xcambar) [inline app/templates/components/builder-field-settings.hbs]: Alright.

the new value does not seem to point to the same value: `bufferDefault` here VS `buffer.defaults.firstObject` in https://github.com/novafloss/formidable-ui/blob/915931eaf8363f672ee0baeaa479225b308cc46c/app/components/builder-field-settings.js#L248

If you confirm this is correct, then 👍
